frontend/src/App.js

```diff
@@ -46,6 +41,23 @@ class App extends Component {
     });
   };
 
+  // Logout state handler function
+  logout = () => {
+    this.setState({
+      sessionId: undefined,
+      showShifts: false
+    });
+  };
+
+  // Get user attributes
+  fetchUserAttributes = sessionId => {
+    return this.fetchRequest(
+      "http://localhost:3000/users/me/",
+      "GET",
```

Comment:
It could be worth having helpers for post/get reqs too, so you can just call `getApi('users/me', sessionId)`